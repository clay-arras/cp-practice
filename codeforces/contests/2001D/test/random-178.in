1
10
20 3 24 18 17 12 4 6 16 24
