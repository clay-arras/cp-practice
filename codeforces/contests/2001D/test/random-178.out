9
20 3 24 18 17 12 4 6 16
